### PR TITLE
Rdrf 2117 sample

### PR DESCRIPTION
**Pull request by jithendralal**
Sample registry yaml and getting started instructions are added as part of the repo.

### CODE REVIEW

File: GETTING_STARTED.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+Please find the Importer link on the admin page.
+
+On the importer page, click Browse button and select the SAMPLE_REGISTRY.yaml file included in the RDRF repo.
+
+This will create a sample registry with a few simple elements, which will help you get started.
+
+Next steps:
+1. From the main admin page, add a WorkingGroup (This will be needed while adding a patient)
+2. From the header navigation menu, select Patient List and add a patient.
+3. You can use Resgistry Common Data Element, Registry Section, Registry Form to create your own data collection pages.
```
Comment (id2359) [inline GETTING_STARTED.txt:16]: typo here ( Resgistry)

File: GETTING_STARTED.txt [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+4. You can use Survey Question, Surveys to create surveys for data collection.
```
Comment (id2359) [inline GETTING_STARTED.txt:17]: Unless we add instructions for setting up a PROMS site not sure we should mention this - better to just have a non-CIC like registry sample - I can add proms system documentation later

**Review by id2359 — CHANGES_REQUESTED** [review]

File: GETTING_STARTED.txt (hunk 1 above)
Comment (jithendralal) [reply GETTING_STARTED.txt]: corrected

File: GETTING_STARTED.txt (hunk 2 above)
Comment (jithendralal) [reply GETTING_STARTED.txt]: removed Survey related step

**Review by id2359 — APPROVED** [review]

**Pull request merged**